walletconsole/lib/WalletConsole.cpp

```diff
@@ -13,9 +13,12 @@
 #include "TonCoin.h"
 
 #include "Base64.h"
+//#include "Base58.h"
 #include "HexCoding.h"
 #include "Data.h"
 
+#include "FIO/TransactionBuilder.h" // TODO
```

Comment:
Is this finished?

Reply:
Ops, I checked in the wrong file.  Corrected.